tests/meta/orders/test_meta_orders_status.py

```diff
@@ -374,6 +374,66 @@ def test_store_families_bad_apptag(orders_api, base_store, mip_status_data):
         )
 
 
+def test_store_case_invalid_capture_kit(orders_api, base_store, balsamic_status_data):
+
+    status_data = balsamic_status_data
+
+    # GIVEN a basic store with no samples or nothing in it + scout order
+    assert base_store.samples().first() is None
+    assert base_store.families().first() is None
+    bed_name = 'non-existing capture kit'
+    assert base_store.bed(bed_name) is None
+
+    for family in status_data['families']:
+        for sample in family['samples']:
+            sample['capture_kit'] = bed_name
+
+    # WHEN storing the order
+        cases = orders_api.store_cases(
+            customer=status_data['customer'],
+            order=status_data['order'],
+            ordered=dt.datetime.now(),
+            ticket=123456,
+            cases=status_data['families'],
+        )
+
+    # THEN it should not have connected any bed to the case
+    assert cases
+    for case in cases:
+        assert case.links
+        for link in case.links:
+            assert not link.sample.bed_version
+
+
+def test_store_case_valid_capture_kit(orders_api, base_store, balsamic_status_data):
+    status_data = balsamic_status_data
+
+    # GIVEN a basic store with no samples or nothing in it + scout order
```

Comment:
Can you double check that the GIVEN - WHEN - THEN descriptions match the test case? They seem to be the same as the test case before.

Reply:
well spotted! fixed now